src/components/styles/Books.css

```diff
@@ -1,11 +1,162 @@
+.bookSection {
+  margin: 0 2%;
+  padding-bottom: 0%;
+}
+
 .books {
   display: flex;
-  gap: 15%;
+  gap: 25%;
+  padding: 1% 2.5%;
+
+  /* width: 90%; */
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  border-radius: 2px;
+  margin: 2%;
+}
+
+.mainDiv {
+  display: flex;
+  flex-direction: column;
+}
+
+.bookCat {
+  opacity: 0.5;
+  font-family: "Montserrat", serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin: 0;
+}
+
+.booksTitle {
+  font-family: serif;
+  font-size: 1.775rem;
+  font-weight: bolder;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -1.2px;
+  color: #121212;
+}
```

Comment:
- [ ] Please 🙏🏽 correct the styles according to the font family specifications to match the Zeplin design.

![image](https://github.com/Atifzada/BookStore_App/assets/58642949/6385f08d-1ece-4367-b96e-067ee9c8e8b3)